cmd/replicate-edit.go

```diff
@@ -54,14 +55,18 @@ var replicateSetFlags = []cli.Flag{
 		Name:  "remote-bucket",
 		Usage: "destination bucket, should be a unique value for the configuration",
 	},
+	cli.StringFlag{
+		Name:  "replicate",
+		Usage: "replicate delete markers, versioned deletes or both (delete, delete-marker)",
```

Comment:
It would be consistent, if we use the same replicate `Usage` info for both `--replicate add` and `--replicate edit` commands.

Here:
`Usage: "replicate delete markers, versioned deletes or both (delete, delete-marker)",`
versus
in replicate-add.go:
`Usage: "comma separated list to enable replication of delete markers, and/or deletion of versioned objects.Valid options are \"delete-marker\", \"delete\"",`